eslintrc.json

```diff
@@ -561,27 +561,47 @@
     {
       "files": "*.vue",
       "globals": {
-        "apos": true,
-        "defineProps": "readable",
-        "defineEmits": "readable",
-        "defineExpose": "readable",
-        "defineOptions": "readable",
-        "defineModel": "readable",
-        "defineSlots": "readable"
+        "apos": "readonly",
+        "defineProps": "readonly",
+        "defineEmits": "readonly",
+        "defineExpose": "readonly",
+        "defineOptions": "readonly",
+        "defineModel": "readonly",
+        "defineSlots": "readonly",
```

Comment:
Do you think it's possible to use `eslint-plugin-vue` 9+?

> ### Compiler macros such as defineProps and defineEmits generate no-undef warnings
> You need to use [vue-eslint-parser] v9.0.0 or later.
> Previously you had to use the vue/setup-compiler-macros environment, this is no longer needed.

source: https://eslint.vuejs.org/user-guide/#compiler-macros-such-as-defineprops-and-defineemits-generate-no-undef-warnings